datastorages/jcloud/src/main/assembly/README.txt

```diff
@@ -0,0 +1,14 @@
+JCloud data storage provider library
+----------------------------------
```

Comment:
I recommend making this file in markdown; and processing to html for end-users.

Reply:
Updated in https://github.com/geonetwork/core-geonetwork/pull/7302/commits/6a37820c57f5ddfab9d46d7409f65066eec937fd